pkg/kademlia/kademlia_test.go

```diff
@@ -150,7 +149,7 @@ func TestBootstrap(t *testing.T) {
 	assert.Len(t, nodeIDs, 3)
 }
 
-func testNode(ctx *testcontext.Context, name string, t *testing.T, bn []pb.Node) (*Kademlia, *grpc.Server, func()) {
+func testNode(ctx *testcontext.Context, t *testing.T, bn []pb.Node) (*Kademlia, *grpc.Server, func()) {
```

Comment:
I think that the `name` parameter shouldn't be removed and it should be used to enhance the assertion error message.

For example:

```go
lis, err := net.Listen("tcp", "127.0.0.1:0")
require.NoErrorf(t, err, "node: %s", name)
```